packages/@jsii/integ-test/test/build-cdk.test.ts

```diff
@@ -44,45 +44,22 @@ describe('Build CDK', () => {
       cwd: srcDir
     });
 
-    // build cdk build tools
-    await processes.spawn('npx', [
-      'lerna',
-      '--stream',
-      '--scope', 'cdk-build-tools',
-      '--scope', 'pkglint',
-      '--scope', 'awslint',
-      'run', 'build',
-    ], { cwd: srcDir });
-
-    // build jsii modules
-    await processes.spawn('npx', [
-      'lerna',
-      '--stream',
-      '--scope', '@aws-cdk/*',
-      '--scope', 'aws-cdk',
-      'run', 'build',
-      '--', '--jsii', path.join(JSII_DIR, 'bin', 'jsii'),
-    ], { cwd: srcDir });
+    // install local version of jsii
+    await processes.spawn('yarn', ['workspace', 'cdk-build-tools', 'add', JSII_DIR, JSII_PACMAK_DIR], {
+      cwd: srcDir
+    });
+    await processes.spawn('yarn', ['add', '-W', JSII_DIR, JSII_ROSETTA_DIR, JSII_PACMAK_DIR], {
+      cwd: srcDir
+    });
 
-    // build the rest
-    await processes.spawn('npx', [
-      'lerna',
-      '--stream',
-      '--ignore', '@aws-cdk/*',
-      '--ignore', 'aws-cdk',
-      '--ignore', 'cdk-build-tools',
-      '--ignore', 'pkglint',
-      '--ignore', 'awslint',
-      'run', 'build',
-    ], { cwd: srcDir });
+    // build cdk modules
+    await processes.spawn('npx', ['lerna', 'run', 'build'], {
+      cwd: srcDir
+    });
 
-    // package modules
-    await processes.spawn('./pack.sh', [], {
-      cwd: srcDir,
-      env: {
-        PACMAK: path.join(JSII_PACMAK_DIR, 'bin', 'jsii-pacmak'),
-        ROSETTA: path.join(JSII_ROSETTA_DIR, 'bin', 'jsii-rosetta')
-      }
+    // package modules with pacmak
+    await processes.spawn('yarn', ['pack'], {
+      cwd: srcDir
     });
```

Comment:
Should we sanity check that the generated pack(s) contain some basic set of artifacts? To prevent this test from succeeding but not actually doing anything.

Reply:
good call out, I will add a step to verify the presence of the various language target directories in `srcDir/dist`